presentation/Main.java

```diff
@@ -0,0 +1,69 @@
+package presentation;
+
+import dao.Dao;
+import dao.StudentDao;
+import model.Student;
+import service.StudentService;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int userOption;
+        Scanner scanner = new Scanner(System.in);
+        Dao<Student> dao = new StudentDao();
+        StudentService studentService = new StudentService(dao);
+        Menu menu = new Menu();
+        do{
+            menu.showMenu();
+            userOption= scanner.nextInt();
+            switch (userOption) {
+                case Menu.ADD_STUDENT -> {
+                    Student studentToAdd = new Student();
+                    System.out.println("Enter the Student Details: ");
+                    System.out.print("Student Id: ");
+                    studentToAdd.setStudentId(scanner.next());
+                    System.out.print("First Name: ");
+                    studentToAdd.setFirstName(scanner.next());
+                    System.out.print("Last Name: ");
+                    studentToAdd.setLastName(scanner.next());
+                    System.out.println();
+                    studentService.addStudent(studentToAdd);
+                    System.out.println();
+                }
+                case Menu.UPDATE_STUDENT -> {
+                    System.out.print("Enter the StudentId to update: ");
+                    String studentId = scanner.next();
+                    Student student = studentService.getStudentById(studentId);
+                    Student student1 = new Student();
+                    System.out.printf("%s%s%s","Enter the studentId (",student.getStudentId(),") : ");
+                    student1.setStudentId(scanner.next());
```

Comment:
The `StudentDao.update` method does not update the `studentId` in the, so the user does not need to be asked about setting a new `studentId`.